package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitfinex-api-node",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
version should be 7.0 because bfx-api-node-rest has breaking changes, otherwise users would just get latest version would have non working code on their integrations

Reply:
The major version has already been bumped in this PR https://github.com/bitfinexcom/bitfinex-api-node/pull/582 which has not been published
And this one, relative to the previous one, contains only a patch

Reply:
true, but it was never released on npm (https://www.npmjs.com/package/bitfinex-api-node)